### DISCUSSION TITLE
Web Searches plugin - google has no default keyword

### DISCUSSION THREAD

**Discussion by deefrawley** [Flow-Launcher/Flow.Launcher#656] [Q&A] [answered]
Was there a design choice for this? Firstly, the animation on the home page shows 'g' being used, and secondly if you do want to google something, the 'search Google' option is usually way down the list of results.

**Comment by jjw24** [comment, marked as answer]
Yeah it's done this way to help beginners to web search more naturally. When beginners download flow they might not even be aware of action keywords, and may randomly type stuff to see what comes up, thought this way instead of those random stuff coming up with nothing, they will be provided at least a search on the web option. 

The gif is out of date for this, but to update just this is a bit of pain(at least when I did it), so maybe someone else can do a better gif preview or we update it at a later point when more things change.

I don't remmeber the score that is set for the Google search, maybe 50? It would be a bit further down the list if there are programs that match, but I suppose it makes sense to present program result first then web search result, i.e. `git` would probably have the Google search result lower than program, but search `git how to` would then place Google search result first.

**Reply by deefrawley** [reply]
No worries. I'm playing around with a fresh install to get some screen grabs for a doco update which is how I cam across it. I might add something about google search being baked in as such and users can assign a hotkey if they want to make a targeted search.

**Reply by taooceros** [reply]
I remember the score is 5
